socialcards/src/ShowCard.jsx

```diff
@@ -2,55 +2,45 @@ import { useEffect, useState } from "react";
 import axios from "axios";
 import { useParams } from "react-router-dom";
 
-const ShowCard = () => {
-    const [frontText, setFrontText] = useState('')
-    const [backText, setBackText] = useState('')
-    const [imageURL, setImageURL] = useState('')
-    const [backgroundColor, setBackgroundColor] = useState('')
-    const [font, setFont] = useState('')
-    const { cardID } = useParams()
-    useEffect(() => {
-        axios
-            .get(`https://social-cards.fly.dev/api/cards/${cardID}/`,
-                { headers: { Authorization: `Token ${token}` } }
-            )
-            .then((response) => {
-                setFrontText(response.data.front_text);
-                setBackText(response.data.back_text)
-                setImageURL(response.data.imageURL)
-                setBackgroundColor(response.data.background_color)
-                setFont(response.data.font)
-            }
-
-            )
-    }, []
-    )
-    return (
-        <>
-            <div>
-                <div>
-                    <img href={imageURL}>
-                    </img>
-                    <div>
-                        {frontText}
-                    </div>
-                    <div>
-                        {backText}
-                    </div>
-                    <div>
-                        {font}
-                    </div>
-                </div>
-            </div>
-        </>
-    )
-}
+const ShowCard = ({ token }) => {
+  const [frontText, setFrontText] = useState("");
+  const [backText, setBackText] = useState("");
+  const [imageURL, setImageURL] = useState("");
+  const [backgroundColor, setBackgroundColor] = useState("");
+  const [font, setFont] = useState("");
+  const { cardID } = useParams();
+  useEffect(() => {
+    axios
+      .get(`https://social-cards.fly.dev/api/cards/${cardID}/`, {
+        headers: { Authorization: `Token ${token}` },
+      })
+      .then((response) => {
+        setFrontText(response.data.front_text);
+        setBackText(response.data.back_text);
+        setImageURL(response.data.imageURL);
+        setBackgroundColor(response.data.background_color);
+        setFont(response.data.font);
+      });
+  }, []);
+  return (
+    <>
+      <div>
+        <div>
+          <img href={imageURL}></img>
+          <div>{frontText}</div>
+          <div>{backText}</div>
+          <div>{font}</div>
+        </div>
+      </div>
+    </>
+  );
+};
 export default ShowCard;
 
 // ${id} for .get `` instead of ''
 // 2 parent divs. 1 with 3 divs other with 4 divs. 2nd parent div doesnt show anything until "click to see caption" is clicked
 // 3 divs
 // A way to capture username for 1st div
 // Showing imageURL for 2nd div
-// click to see caption 3rd div with onclick function (inside a ternary statement) 
-// edit card
\ No newline at end of file
+// click to see caption 3rd div with onclick function (inside a ternary statement)
+// edit card
```

Comment:
are these changes to ShowCard just formatting? It looks like it to me -- looks like @WilkinsJay you're indenting to 4 spaces (or 2 tabs) and @CRiddleNC you have 2 spaces (or a tab). You all should be indenting to the same number of spaces so that you don't have these diffs that are only about reformatting. This will be easier if everyone is using the same formatter -- make sure you have Prettier installed in VS Code. https://momentum-team-19.github.io/course/phase3-fe/intro-react/#%EF%B8%8F-setup

If this is just a formatting diff and no code changes, @CRiddleNC can you remove this change from the PR?

Reply:
Gavin and I have the 2 tab, so we should make this change on Jaylans VS Code.